source/validation.txt

```diff
@@ -35,14 +35,14 @@ specific shape or only allowing a specified range of values in fields.
 Validation Rules
 ----------------
 
-*Updated in version 1.18.*
+*Updated in version 1.35.1*
 
 The validation editor supports :manual:`JSON Schema validation
 </core/schema-validation/#json-schema>`, and validation with query
-expressions using :manual:`query operators
-</reference/operator/query>`. As you edit your validation,
-|compass-short| updates in real-time to display a document from your
-collection that passes the validation and a document that fails.
+expressions using :manual:`query operators </reference/operator/query>`.
+When you hit the Update button, Compass updates to display a document
```

Comment:
[nit] For UI instructions, we want to be more explicit in what the users have to do — it might be confusing for international audiences to read about "hitting" a button.

Also for the update button, we want to add the `:guilabel:` directive!
```suggestion
After you click the :guilabel:`Update` button, Compass updates to display a document
```

Reply:
Done! Thanks for your feedback!